### PR TITLE
doc: Add documentation for ad5770r, ad5933, ltc268x console projects

### PR DESCRIPTION
**Pull request by SaikiranGudla**
Added sphinx documentation and updated TOC using index.rst files

### CODE REVIEW

**Comment by SaikiranGudla** [comment]
Hi all,
Any comments on this PR?

File: doc/sphinx/source/projects/ad5770r_console/ad5770r_console.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      :width: 500
+
+The AD5770R Evaluation Board is connected to SDP-K1 using the 120-pin SDP Connector. The evaluation board must be powered externally, typically using an external 3.3V DC supply connected to the AVDD input supply terminal, if using the default power supply jumper settings.
+
+The SDP-K1 is connected to a PC through a USB cable. The firmware image, a .BIN file created using the online mbed compiler, can be loaded into SDP-K1 board by copying the .BIN file to the drive representing the SDP-K1. The SDP-K1 provides a USB to Serial link (UART) and any serial terminal (such as Teraterm, Putty, Coolterm, etc) can be used to connect to by configuring terminal with the necessary settings (serial port, baud rate, data bits, etc).
+
+.. note::
+   Connect the VIO_ADJUST jumper on the SDP-K1 board to 3.3V position to drive SDP-K1 GPIOs at 3.3V
+
+.. Useful links Section
```
Comment (D-Disha) [inline doc/sphinx/source/projects/ad5770r_console/ad5770r_console.rst:55]: Can we add an extra line below, same for other projects as well

File: doc/sphinx/source/projects/ad5770r_console/ad5770r_console.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+* Follow the steps mentioned in the Build Guide section above.
+* Start up a serial terminal emulator (e.g. Tera Term)
+   * Find the com-port your controller board is connected on and select it.
+   * Set the baud-rate for 230400
+   * Reset the controller board and connect.
+* The terminal windows should display the console menu providing access to the AD5770R functionality.
+
+==================
+Using the Firmware
+==================
```
Comment (D-Disha) [inline doc/sphinx/source/projects/ad5770r_console/ad5770r_console.rst:80]: Can we add an extra line below

File: doc/sphinx/source/projects/ad5933_console/ad5933_console.rst [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+.. image:: /source/projects/ad5933_console/ad5933_hardware_connections.jpg
+   :width: 500
+
+.. note::
+    One thing to note here is that power and ground for the evaluation need to be provided and can be conveniently taken from the Arduino header as shown above. If using a different evaluation board to the DIGILENT PMOD IA, then consult the relevant evaluation board guides available through the product-page for your selected board.
+
+====================
+AD5933 Mbed Firmware
+====================
```
Comment (D-Disha) [inline doc/sphinx/source/projects/ad5933_console/ad5933_console.rst:62]: Can we add extra line below

**Comment by SaikiranGudla** [comment]
Change log: Addressed review comments

Added extra lines as required.

**Review by jansunil — APPROVED** [review]

**Pull request merged**